### PR TITLE
Adds Examples and Begins Refactoring

### PR DESCRIPTION
**Pull request by garrettdimon**
This starts adding examples and creating reusable bits where handy.

An example example...
2024-02-01-15-03-39-Safari@2x

### CODE REVIEW

**Review by bkeepers — APPROVED** [review]
👍 looking good!

File: app/models/example.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+class Example
```
Comment (jnunemaker) [inline app/models/example.rb:1]: 😎

File: app/views/example/_circuit_breakers.html.erb [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+]) %>
+
+
+<% content_for :aside do %>
+  <div class="card border-5 border-dark rounded-4">
+    <div class="card-header p-3 pb-2 border-bottom">
+      <%= flag_status(@example.flag) %>
+      <ul class="list-group list-group-flush pt-0 border-top mt-2">
+        <% [
+          ['Fully enabled for everyone?', Flipper.enabled?(@example.flag), "Flipper.enabled?(:#{@example.flag})"],
```
Comment (jnunemaker) [inline app/views/example/_circuit_breakers.html.erb:14]: I'd be tempted to make @example.enabled?:

```ruby
def enabled?(...)
  Flipper.enabled?(flag, ...)
end
```

Comment (jnunemaker) [reply app/views/example/_circuit_breakers.html.erb:14]: Downside is it tucks the Flipper syntax away. Upside is it keeps this clean and shows how we would probably really use it IRL. Either way is fine, just a thought.

**Review by jnunemaker — APPROVED** [review]
Just skimmed but Brandon already approved anyway. 😄

**Pull request merged**